lib/elixir/src/elixir_lexical.erl

```diff
@@ -44,6 +44,9 @@ record_alias(Module, Ref) ->
 record_import({Module, Function, Arity}, Ref) ->
```

Comment:
Let's see if we can get rid of this one per above given the new clause below. :)